components/browser/engine-system/src/main/java/mozilla/components/browser/engine/system/SystemEngineSession.kt

```diff
@@ -253,6 +254,19 @@ class SystemEngineSession(private val defaultSettings: Settings? = null) : Engin
         // no-op
     }
 
+    override fun captureThumbnail(): Bitmap? {
+        val webView = currentView()
+
+        return if (webView != null) {
```

Comment:
Should use idiomatic Kotlin here? webview?.let {}

Reply:
👍